ferrilata-devilline/Properties/launchSettings.json

```diff
@@ -1,17 +1,28 @@
 {
-  "profiles": {
-    "ferrilata_devilline": {
-      "commandName": "Project",
-      "launchBrowser": true,
-      "environmentVariables": {
-        "FDUSERNAME": "root",
-        "FDDATABASE": "ProjectDB",
-        "FDPASSWORD": "",
-        "FDHOST": "localhost",
-        "FDTOKENSECRET": "6c07d2e1-3395-4780-8c4a-900f58202043",
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      },
-      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    "iisSettings": {
+        "windowsAuthentication": false,
+        "anonymousAuthentication": true,
+        "iisExpress": {
+            "applicationUrl": "http://localhost:56788",
+            "sslPort": 44329
+        }
+    },
+    "profiles": {
+        "IIS Express": {
+            "commandName": "IISExpress",
+            "launchBrowser": true,
+            "environmentVariables": {
+                "ASPNETCORE_ENVIRONMENT": "Development",
+                "FDTOKENSECRET": "6c07d2e1-3395-4780-8c4a-900f58202043"
+            }
+        },
+        "ferrilata_devilline": {
+            "commandName": "Project",
+            "launchBrowser": true,
+            "environmentVariables": {
+                "ASPNETCORE_ENVIRONMENT": "Development"
+            },
```

Comment:
Please add FDTOKENSECRET to "ferrilata_devilline" profile as well, otherwise it doesn't run properly on a Mac

Reply:
done